http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeoException.scala

```diff
@@ -24,10 +24,12 @@ class LeoException(val message: String = null,
 final case class BadRequestException(msg: String, traceId: Option[TraceId])
     extends LeoException(msg, StatusCodes.BadRequest, traceId = traceId)
 
-final case class AuthenticationError(email: Option[WorkbenchEmail] = None)
-    extends LeoException(s"${email.map(e => s"'${e.value}'").getOrElse("Your account")} is not authenticated",
-                         StatusCodes.Unauthorized,
-                         traceId = None)
+final case class AuthenticationError(email: Option[WorkbenchEmail] = None, extraMessage: String = "")
```

Comment:
Is `extraMessage` used?

Reply:
yea, I'm using it in https://github.com/DataBiosphere/leonardo/pull/2689/files#diff-4bbc2634fc8c5c714a0b57a58781db11b31911b90e711cfd6051f5572edb1c6eR133 to log a bit more info...but it's not returned to user

Reply:
I see 👍